Bugfix/pet structure and store

okhabr opened the pull request:
Structural changes:
- Remove redundant components
- Update store folder
- Add TODOs in my component
- Update route so pet page is fixed

okhabr committed:
Merge branch 'bug-fix/Pet_Profile' into develop

okhabr committed:
Merge branch 'develop' of https://github.com/IvanBonk/snow-project into develop

okhabr committed:
Merge branch 'develop' of https://github.com/IvanBonk/snow-project into develop

okhabr committed:
Minor changes

okhabr committed:
Merge branch 'develop' of https://github.com/IvanBonk/snow-project into develop

okhabr committed:
Structural changes:
- Remove redundant components
- Update store folder
- Add TODOs in my component

enheit commented on src/pages/pet/pet.component.tsx:
```diff
@@ -1,9 +1,9 @@
 import React, { useEffect, Fragment } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
-import { fetchPetProfile } from '../../store/actions/pet.action';
+import { fetchPetProfile } from '../../store/pet/action/pet.action';
```
Name `action` folder in plural form. I think it will contain more than one action >_<

okhabr replied:
done

enheit commented on src/store/pet/models.ts:
```diff
@@ -1,11 +1,10 @@
-import { PetProfile } from '../shared/models';
+import { PetProfile } from '../../shared/models';
```
Wouldn't be better to rename `src/store/pet/models.ts` file to `src/store/pet/pet.models.ts`?

okhabr replied:
it would:)

Danik1702 reviewed (approved):
Great!

okhabr committed:
Modify as per Roma's comments:
 - rename files&folder
 - minor Observations change - remove redundant words in template
 - remove any type from saga

okhabr committed:
Merge branch 'develop' into bugfix/pet-structure-and-store
